src/main/webapp/app/complaints/complaints-for-tutor/complaints-for-tutor.component.ts

```diff
@@ -51,6 +52,12 @@ export class ComplaintsForTutorComponent implements OnInit {
     ngOnInit(): void {
         this.course = getCourseFromExercise(this.exercise!);
 
+        this.maxComplaintResponseTextLimit = this.course!.maxComplaintResponseTextLimit ?? 0;
+        if (!this.exercise?.course) {
```

Comment:
Would it be better to check for the existence of the exercise group instead of checking if there is no course?

Reply:
I think functionality wise it should not make a difference, but it probably makes the code a bit more readable. 👍